test/automation/src/sql/notebook.ts

```diff
@@ -13,15 +13,18 @@ export class Notebook {
 
 	public readonly notebookToolbar: NotebookToolbar;
 	public readonly textCellToolbar: TextCellToolbar;
+	public readonly notebookFind: NotebookFind;
 	public readonly view: NotebookTreeView;
 
-	public readonly winOrCtrl = process.platform === 'darwin' ? 'ctrl' : 'win';
 	public readonly ctrlOrCmd = process.platform === 'darwin' ? 'cmd' : 'ctrl';
+	public readonly winOrCtrl = process.platform === 'darwin' ? 'ctrl' : 'win';
```

Comment:
Let's take the opportunity to move these out - they're more generic than just Notebooks. I'd suggest making a new `test/automation/src/sql/constants.ts` to put these kind of values in. 

(and then remove the duplicated ctrlOrCmd below)